scripts/load_incidents_v3.py

```diff
@@ -121,6 +121,7 @@ def create_schema(schema_path, api, headers=None):
                              data={'label': 'Incident',
                                    'plural_label': 'Incidents',
                                    'description': 'Historical incident data',
+                                   'temporal': True,
                                    'active': True},
                              headers=headers)
     response.raise_for_status()
```

Comment:
Nothing you introduced, but I noticed when testing this that the data for the events that are created doesn't quite line up with the schema, which is causing the record list and record editors to show blanks for all the fields. It's a trivial fix though: Lines 51-53 need to have their keys changed to match what is in `incident_schema_v3.json`. This should be as simple as prefixing with `driver` and camel-casing, i.e.:
- `incidentDetails` -> `driverIncidentDetails`
- `vehicle` -> `driverVehicle`
- `person` -> `driverPerson`

Reply:
I changed this, though it appears some attributes under `driverIncidentDetails` fail Grout's type-based validation (e.g., `u'0'` fails `number` validation). I added a typecast function to the `details_mapping` list, which allows that to pass but that revealed that some of the data appears to be legitimately invalid (`u'11' is not one of [u'Head on', u'Rear end', u'Right angle', u'Side swipe', u'Overturned vehicle', u'Hit object in road', u'Hit object off road', u'Hit parked vehicle', u'Hit pedestrian', u'Hit animal']`). I'm going to investigate the origin of this and see if it's feasible to fix or exclude invalid rows, or if I should drop the bad column from the sample data entirely.

Reply:
This should be good now, changed my anonymization script to exclude rows that were missing lat/lon coordinates or had numeric or blank values for required string parameters. Filtered out a good almost 20% of the records, but everything we have now should be schema appropriate.